docs/shipping/Code/http.md

```diff
@@ -138,5 +138,173 @@ Allow some time for data ingestion, then open [Open Search Dashboards](https://a
 
 Encounter an issue? See our [log shipping troubleshooting](https://docs.logz.io/docs/user-guide/log-management/troubleshooting/log-shipping-troubleshooting/) guide.
 
+</TabItem>
+<TabItem value="http-otlp" label="Protobuf via OpenTelemetry" default>
+
+This guide provides step-by-step instructions to Logz.io users on how to send logs in Protobuf format using the OTLP listener. Follow these steps to set up your environment and send logs via the OTLP protocol using the protocurl tool.
+
+## Download `protocurl`
+
+`protocurl` is a tool based on curl and Protobuf, designed for working with Protobuf-encoded requests over HTTP. Follow the instructions on the [`protocurl` GitHub page](https://github.com/qaware/protocurl) to download and install the tool on your machine.
+
+Once installed, verify the installation with:
+
+```bash
+protocurl --version
+```
+
+## Download OpenTelemetry Protobuf Definitions
+
+Download the OpenTelemetry Protobuf definitions from the [OpenTelemetry-proto GitHub repository](https://github.com/open-telemetry/opentelemetry-proto/tree/main).
+
+You need the `.proto` files to compile Protobuf messages and send logs. Download the repository to a local folder, for example:
+
+```bash
+git clone <https://github.com/open-telemetry/opentelemetry-proto.git> ~/Downloads/proto/opentelemetry-proto
+```
+
+## Prepare the Command
+
+Once `protocurl` is installed and the OpenTelemetry Protobuf files are downloaded, you can start sending logs. 
+
+Here is the basic structure of the command:
+
+```bash
+protocurl -v \\
+  -I ~/Downloads/proto/opentelemetry-proto \\
+  -i opentelemetry.proto.collector.logs.v1.ExportLogsServiceRequest \\
+  -o opentelemetry.proto.collector.logs.v1.ExportLogsServiceResponse \\
+  -u '<https://otlp-listener.logz.io/v1/logs>' \\
+  -H 'Authorization: Bearer <Logzio-Token-Logs>' \\
+  -H 'user-agent: logzio-protobuf-logs' \\
+  -d @export_logs_request.json
+```
+
+Breakdown:
+
+* `I`: Points to the location of the OpenTelemetry Protobuf definitions.
+* `i`: Specifies the Protobuf request type (`ExportLogsServiceRequest`).
+* `o`: Specifies the Protobuf response type (`ExportLogsServiceResponse`).
+* `u`: URL of the Logz.io OTLP listener endpoint. Adjust the URL for your region using Logz.io [region settings](https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/).
```

Comment:
Lets refer to the relevant region table:
https://docs.logz.io/docs/user-guide/admin/hosting-regions/account-region/#opentelemetry-protocol-otlp-regions